### PR TITLE
Tweaked example code. Tested on Feather M4.

### PR DESCRIPTION
**Pull request by kattni**

### CODE REVIEW

**Comment by caternuson** [comment]
Look like mainly cosmetic changes to the example. What was the motivation for this?

**Comment by kattni** [comment]
It is mostly cosmetic. Motivation is a complete redo of the guide to match the way we're doing it for the Python and CircuitPython usages, so I wanted the example to be more in line with the updated usage explanation.

File: examples/ccs811_simpletest.py [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-temp = ccs.temperature
-ccs.temp_offset = temp - 25.0
+temp = ccs811.temperature
+ccs811.temp_offset = temp - 25.0
 
 while True:
-    print("CO2: ", ccs.eco2, " TVOC:", ccs.tvoc, " temp:", ccs.temperature)
-    time.sleep(.5)
+    print("CO2: %1.0f PPM, TVOC: %1.0f PPM, Temp: %0.1f C" %
+          (ccs811.eco2, ccs811.tvoc, ccs811.temperature))
```
Comment (caternuson) [inline examples/ccs811_simpletest.py]: Should we use `.format` here instead of `%`? I think `%` is considered the "old" way.

**Review by caternuson — COMMENTED** [review]
Just a few comments.

File: examples/ccs811_simpletest.py [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
-from board import SCL, SDA
+import board
 import busio
-
 import adafruit_ccs811
 
-i2c_bus = busio.I2C(SCL, SDA)
-
-ccs = adafruit_ccs811.CCS811(i2c_bus)
+i2c_bus = busio.I2C(board.SCL, board.SDA)
```
Comment (caternuson) [inline examples/ccs811_simpletest.py]: `i2c_bus` or just `i2c`?

**Review by brennen — APPROVED** [review]

File: examples/ccs811_simpletest.py (hunk 1 above)
Comment (kattni) [reply examples/ccs811_simpletest.py]: Hmm I didn't realise it was considered to be the old way. I'll refactor it.

File: examples/ccs811_simpletest.py (hunk 2 above)
Comment (kattni) [reply examples/ccs811_simpletest.py]: I had `i2c`. Must have pasted the wrong update at some point in going back and forth between code.py and simpletest.

**Review by caternuson — APPROVED** [review]

**Pull request merged**